test/features/perform_sanity_checks_on_reviews_test.rb

```diff
@@ -0,0 +1,150 @@
+require 'test_helper'
+Dir[Rails.root.join('test/**/*.rb')].each { |f| require f }
+
+class PerformsSanityChecksOnReviewsTest < Capybara::Rails::TestCase
```

Comment:
I also like that you're adding these tests. I think the comments are quite helpful too.